### ISSUE TITLE
Display Results Slow Performance

### ISSUE THREAD

**Issue by JASM2007** [microsoft/azuredatastudio#7186]
- Azure Data Studio Version:
![image](https://user-images.githubusercontent.com/40634806/64796227-62e6c600-d54d-11e9-91ba-a9f5829a19a5.png)

I am currently running the latest update as of today.

Steps to Reproduce:

1. I wrote one query that had inner joins and left joins which has about 40 or so fields and then did SELECT * to do a bench-marking test between SSMS and Azure Data Studio.  
2. Run the same query on both programs.
3. SSMS took 1 minute and 47 to execute.  In Azure Data Studio it took 2 minutes and 27 seconds to complete.  That's a 40 second difference.  
4. I then performed the same test but then did a SELECT COUNT(*) instead of the SELECT * and the speed was about the same or to fast to notice a difference.
5. It took about 1 minute and 25 seconds for the results pane to display the values in each field. Whereas SSMS displays the first few records immediately.
6. I did another test and when you do SELECT TOP 100000 then there was a difference of 9 seconds in SSMS and 14 seconds in Azure Data Studio. So it seems like once you start displaying 100000 records then performance starts play a factor.

I hope my tests help!

This difference leads me to believe the issue must have something to do with the results pane and populating the display results in the program.  There may need to be some optimization or performance tweaking to up that speed if it's possible. In talking with coworkers and the same theme keeps appearing with Azure Data Studio is nice but it's slower than SSMS by a fairly significant margin. To the point if I need to display either a large amount of records or a large amount of fields I don't bother running on Data Studio as I know SSMS will complete it faster.

Can we please look into speeding this process up if possible assuming it's not a technological issue?

**Comment by ek1977** [comment]
It seems like this could basically be fixed if Data Studio started populating the results as they come in, do the first 50 rows as soon as they come in then the rest in maybe blocks of 1000.  That way you can see if you are getting the expected results right away instead of having to wait for all of the data to load.

**Comment by VelizarVESSELINOV** [comment]
I agree is very slow (between 40 seconds and more than 1 minute), compared to SSMS (~17 seconds).
Faster the Azure Web Query: 9 minutes.

SSMS not available on Mac, so only Azure Data Studio or the web available as possible options.

Side note:
Very slow compared to web Google BigQuery: 17 seconds (first run), 0 seconds (second run, due to the free 24 hours cache)
